hail/python/hailtop/utils/validate/validate.py

```diff
@@ -178,3 +198,7 @@ def numeric(**conditions: Callable[[Any], Any]):
 
 def switch(key: str, checkers: Dict[str, Dict[Key, Validator]]):
     return SwitchValidator(key, checkers)
+
+
+def anyof(*checkers: Validator):
```

Comment:
nice addition; I wonder if we can think of words that are more obviously different from one another than "oneof" and "anyof". Maybe "isoneof" and "anyof"? The "is" makes me think of python's `is`.